truss/util/path.py

```diff
@@ -75,3 +78,45 @@ def build_truss_shadow_target_directory(stub: str, truss_path: Path) -> Path:
     target_directory_path = Path(Path.home(), ".truss", "models", f"{stub}-{suffix}")
     target_directory_path.mkdir(parents=True, exist_ok=True)
     return target_directory_path
+
+
+def load_gitignore_patterns(gitignore_file: Path):
```

Comment:
nit: these ignore patterns are independent of git – maybe rename to the references to `TRUSS_IGNORE_PATH` and`load_trussignore_patterns` for clarity